org.atnos.site.OutOfTheBox.html

```diff
@@ -428,7 +428,7 @@ <h3 id="async">Async</h3>
 </code></pre>
 <p>Then we need an interpreter to interpret them as <code class="prettyprint">Futures</code>. The <code class="prettyprint">AsyncFutureInterpreter.create</code> method uses an implicit <code class="prettyprint">ExecutionContext</code> to create the interpreter. If we import the interpreter methods we can finally invoke <code class="prettyprint">runAsyncFuture</code> to get a <code class="prettyprint">Future</code> back.</p>
 <pre><code class="prettyprint">
-val interpreter = AsyncFutures.create
+val interpreter = AsyncFutureInterpreter.create
```

Comment:
This actually reflects the current state of the library; the other docs are the ones in need of update (`AsyncTaskInterpreter` is now `AsyncTasks`, etc.).